### PR TITLE
Remove reference to 412 for policy/business-logic

### PR DESCRIPTION
**Pull request by karcass**
Such mechanisms will be developed in different endpoint(s) or API(s)

### CODE REVIEW

**Comment by rf-** [comment]
Should this also remove the `inactive`, `unavailable`, `no_active_trip`, and `trip_not_complete` 400 responses if the idea is to stop trying to validate the state machine at request time?

**Comment by karcass** [comment]
@rf- yeah, good catch.  PR updated.

**Comment by toddapetersen** [comment]
I agree...  412 should be removed.

File: agency/README.md [hunk 1]
```diff
@@ -203,7 +198,7 @@ List of valid vehicle events and the resulting vehicle status if the event is su
...
 | `trip_start`           | Customer starts a trip                                                                               | `available`, `reserved`                          | `trip`              |                                                                         |
 | `trip_enter`           | Customer enters a service area managed by agency during an active trip.                              | `unavailable`, `removed`, `elsewhere`              | `trip`              |                                                                         |
-| `trip_leave`           | Customer enters a service area managed by agency during an active trip.                              | `trip`                                             | `elsewhere`         |                                                                         |
+| `trip_leave`           | Customer leaves a service area managed by agency during an active trip.                              | `trip`                                             | `elsewhere`         |                                                                         |
```
Comment (lionel-panhaleux) [inline agency/README.md]: Maybe table description should be edited too ?
```
List of valid vehicle events and the resulting vehicle status if the event is sucessful.
```
→ 
```
List of valid vehicle events and the resulting vehicle status. `initial state` values are only indicated for convenience.
```

**Comment by lionel-panhaleux** [comment]
Sorry for the delay, 👍 for me.
A rebase & some changes may be needed since https://github.com/CityOfLosAngeles/mobility-data-specification/pull/258 has been merged.

**Pull request merged**